lib/node_modules/@stdlib/assert/is-ragged-nested-array/test/test.js

```diff
@@ -0,0 +1,89 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-unused-vars */
```

Comment:
Test files should never have unused variables.

Reply:
That's correct, thanks! Also, just noticed that the package 'is-array-like-object', had this line in the test.js file. 
`/* eslint-disable object-curly-newline, no-unused-vars */
`
I'm not sure if it's the expected behavior there, but in case it isn't, just letting you know!